src/pke/include/schemebase/base-pre.h

```diff
@@ -84,7 +84,7 @@ class PREBase {
    */
   virtual Ciphertext<Element> ReEncrypt(
       ConstCiphertext<Element> ciphertext, const EvalKey<Element> evalKey,
-      const PublicKey<Element> publicKey = nullptr, usint noiseflooding = 0) const;
+      const PublicKey<Element> publicKey = nullptr) const;
```

Comment:
no default values for virtual function parameters.

Reply:
I don't see this anymore, I had already removed it. Not sure if you were looking at a previous commit?

Reply:
look again. i don't know what revision you look at: https://github.com/openfheorg/openfhe-development/blob/18cf64f6c6af620088c193ff1a31e723a62fbcbc/src/pke/include/schemebase/base-pre.h#L87

Reply:
I was referring to usint noiseflooding = 0 which isn't there anymore. I can remove the default for publickey